source/retryable-writes/tests/README.rst

```diff
@@ -296,6 +296,41 @@ supported write operations:
   - ``bulkWrite()`` with ``ordered=true`` (no ``UpdateMany`` or ``DeleteMany``)
   - ``bulkWrite()`` with ``ordered=false`` (no ``UpdateMany`` or ``DeleteMany``)
 
+Prose Tests
+===========
+
+The following tests ensure that retryable writes work properly with replica sets
+and sharded clusters.
+
+#. Test that retryable writes raise an exception when using the MMAPv1 storage engine,
+   available in both MongoDB 3.6 and 4.0.
+
+   .. code:: java
```

Comment:
I think you should write this in prose rather than Java.  Especially the assertions are going to look very different in other languages (which don't all have exception chaining).

Reply:
Converted the Java code to prose.